infra/outputs.tf

```diff
@@ -39,7 +41,7 @@ data:
     .:53 {
         errors
         health
-        rewrite name database.local ${module.db.db_hostname}
+        rewrite name database.local ${var.db_hostname}
```

Comment:
Would something like `rewrite name database.local ${var.db_instance_enabled ? module.db.db_hostname : var.db_hostname}` at least preserve the existing output when enabled?

Reply:
Won't work directly like that since one is a string and the other a list (given the counts).
Which also means I have probably introduced another bug which hadn't been picked up in my testing regime when the db is enabled.

Working on it...

Reply:
Fixed with ugly hack you supplied